src/site/content/en/secure/fixing-mixed-content/index.md

```diff
@@ -0,0 +1,240 @@
+
+{% Aside %}
+The [report-uri](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri) response header is being deprecated in favor of
+[report-to](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-to).
+Browser support for `report-to` is currently limited to Chrome and Edge.
+You can provide both headers, in which case `report-uri` will be ignored if the browser supports `report-to`.
+{% endAside %}
+
+Whenever a user visits a page on your site,
+their browser sends JSON-formatted reports regarding anything that violates the content security policy to
+`https://example.com/reportingEndpoint`.
+In this case, anytime a subresource is loaded over HTTP, a report is sent.
+These reports include the page URL where the policy violation occurred and the subresource URL that violated the policy.
+If you configure your reporting endpoint to log these reports,
+you can track the mixed content on your site without visiting each page yourself.
+
+The two caveats to this are:
+
+* Users have to visit your page in a browser that understands the CSP header. This is true for most modern browsers.
+* You only get reports for pages visited by your users.
+So if you have pages that don't get much traffic,
+it might be some time before you get reports for your entire site.
+
+The [Content security policy](https://developers.google.com/web/fundamentals/security/csp/)
+guide has more information and an example endpoint.
+
+### Alternatives to reporting with CSP
+
+If your site is hosted for you by a platform such as Blogger,
+you may not have access to modify headers and add a CSP.
+Instead, a viable alternative could be to use a website crawler to find issues across your site for you, such as
+[HTTPSChecker](https://httpschecker.net/how-it-works#httpsChecker)
+or
+[Mixed Content Scan](https://github.com/bramus/mixed-content-scan).
+
+### Upgrading insecure requests
+
+Browsers are beginning to upgrade and block insecure requests.
+You can use CSP directives to force automatic upgrading or blocking of these assets.
+
+The [`upgrade-insecure-requests`](https://www.w3.org/TR/upgrade-insecure-requests/)
+CSP directive instructs the browser to upgrade insecure URLs before making network requests.
+
+As an example, if a page contains an image tag with an HTTP URL such as
+`<img src="http://example.com/image.jpg">`
+
+The browser instead makes a secure request for
+`https://example.com/image.jpg`, thus saving the user from mixed content.
+
+You can enable this behavior either by sending a `Content-Security-Policy` header with this directive:
+
+```markup
+Content-Security-Policy: upgrade-insecure-requests
+```
+
+Or by embedding that same directive inline in the document's `<head>`
+section using a `<meta>` element:
+
+```html
+<meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
+```
+
+As with browser automatic upgrading, if the resource is not available over HTTPS,
```

Comment:
Optional: One difference between Chrome's automatic upgrades and the upgrade-insecure-requests is that Chrome's autoupgrade mechanism only attempts to upgrade active MC, while upgrade-insecure-requests will attempt to upgrade all, including active mixed content. It might be worth highlighting the difference in this section (though I'm also ok leaving this as is if you think that will add more noise than signal, considering active mixed content is fairly rare, and blocked anyways).